tests/mq_llm_engine/test_abort.py

```diff
@@ -0,0 +1,67 @@
+"""Test that aborting is handled properly."""
+
+import asyncio
+import tempfile
+import uuid
+
+import pytest
+
+from tests.mq_llm_engine.utils import RemoteMQLLMEngine, generate
+from vllm.engine.arg_utils import AsyncEngineArgs
+
+MODEL = "google/gemma-1.1-2b-it"
+ENGINE_ARGS = AsyncEngineArgs(model=MODEL)
```

Comment:
can we use dummy weights to accelerate loading? looks like we don't need the actual mode here.